### PR TITLE
Week 10 - Sofie Pellegrini 

### PR DESCRIPTION
**Pull request by SofiePellegrini**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/index.css [hunk 1]
```diff
@@ -1,13 +1,191 @@
...
+    width: 50vw;
+  }
+
+  .btn {
+    width: 10vw;
+    margin-top: 30px;
+  }
+
+  .btn:hover {
+    box-shadow: 0 0px 50px 0 rgb(255, 255, 255), 0 0px 50px 0 rgb(255, 255, 255);
```
Comment (EmmaaBerg) [inline code/src/index.css:189]: Nice detail of the buttons, matches the theme really well :D

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+import Form from 'Components/Form'
 
 export const App = () => {
   return (
     <div>
-      Find me in src/app.js!
+     <Form />
     </div>
   )
 }
```
Comment (EmmaaBerg) [inline code/src/App.js:10]: You have a very clean entry point which is good!

File: code/src/Components/ThirdQuestion.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            <label className='dormitory'>
+              <input
+                type='radio'
+                name='dormitorys'
+                value='Slytherin'
+                onChange={onHomeInputChange}
+                defaultChecked={homeInput === 'Slytherin'}
+              />
+              Slytherin
+            </label>
```
Comment (EmmaaBerg) [inline code/src/Components/ThirdQuestion.js:19]: A suggestion for projects in the future is to create an array with the different options and then map through them, in that way you don't have to repeat yourself so much :D Something like
`const homeArray = ['Slytherin', 'Ravenclaw', 'Gryffindor', 'Hufflepuff']` and then use `{homeArray.map(() => (
))}` to go through the array.

**Review by EmmaaBerg — APPROVED** [review]
Really good work with this project. Your survey fulfills the requirements and I think it was easy to follow with the different components. The code follows the guidelines on how to write good code as well and is responsive as far as I can see. Left a comment on the radio buttons for future projects about DRY, but that's just a detail, and the more we learn about code the more natural it will be I guess :D

File: code/src/index.css [hunk 4]
```diff
@@ -1,13 +1,191 @@
...
+.slider::-moz-range-thumb {
+  width: 25px;
+  height: 25px;
+  border-radius: 50%;
+  background: url(assets/harryslider.png);
+  cursor: pointer;
+}
+
+.input[type='range']::ms-fill-lower,
+.input[type='range']::ms-fill-upper {
```
[the remark this answers is not in the document]
Comment (SofiePellegrini) [reply code/src/index.css:129]: When I tested my project in different browsers I noticed that it looked really of in Explorer so these are for styling it in Explorer. Even though it seems impossible to get it completely right there.. :)

File: code/src/Components/ThirdQuestion.js (hunk 3 above)
Comment (SofiePellegrini) [reply code/src/Components/ThirdQuestion.js:19]: Thanks for the tip! Didn't think about that :D

**Pull request closed without merging**